src/Tasks/NativeMethods.cs

```diff
@@ -57,7 +57,7 @@ public interface IFixedTypeInfo
     [GuidAttribute("00020406-0000-0000-C000-000000000046")]
     [InterfaceTypeAttribute(ComInterfaceType.InterfaceIsIUnknown)]
     [ComImport]
-    internal interface UCOMICreateITypeLib
+    internal interface IUCOMICreateITypeLib
```

Comment:
```suggestion
    internal interface ICreateTypeLib
```

The `UCOM` prefix is obsolete and the extra `I` was likely not intended in the first place. The interface as defined in `oaidl.idl` is named `ICreateTypeLib`.